### PR TITLE
Add context summary for week 7

### PR DESCRIPTION
**Pull request by marisid**
Related to #33

### CODE REVIEW

**Comment by msachi** [comment]
@marisid Well written, and I think all of this content should be presented to them. I just wonder if all of it belongs here - it might be better to stick to just what is Hapi and Handlebars, and then cover the ideas of frameworks & libraries in the Mon morning 'talk'?

**Comment by msachi** [comment]
@bradreeder Do you have an opinion on what the summary / context should look like?

File: coursebook/week-7/summary.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Week 7
+This week you will be introduced to a Node framework, called *hapi.js* and to a JavaScript templating engine, called *Handlebars.js*. You will then write your own hapi server and build a blog site using a postgreSQL database and server-side template rendering.
+
+As this is the first week when you are introduced to a framework, it's worth saying a few words about abstractions/libraries and fraweworks before diving into hapi.js.
```
Comment (bradreeder) [inline coursebook/week-7/summary.md]: typo "frameworks"

**Comment by bradreeder** [comment]
@msachi I'd originally intended to have the summary (or monday intro talk) in the learning outcomes file, only as I figured they're delivered alongside each other. In honesty having it in a separate summary.md file makes sense. This is fine by me for now, as we can always edit this to conform with the prior weeks structures as we go.

In terms of content, if this is the talk you're delivering at the start of the week then I think it should look like whatever you're comfortable delivering. I doubt each of the weeks coursebooks are going to look identical unless a single editor, like myself, takes a heavy involvement.

**Comment by msachi** [comment]
Thanks @bradreeder. In that case, @marisid I'm happy for this to stay as it is (just fix the typo) for now, and when I'm preparing my Monday morning intro I might add to it or modify it slightly.

**Comment by marisid** [comment]
@maschi typo fixed - merge when you can :smile:

**Pull request merged**